core/src/mindustry/entities/bullet/BulletType.java

```diff
@@ -144,6 +144,9 @@ public boolean collides(Bullet bullet, Building tile){
     }
 
     public void hitTile(Bullet b, Building tile, float initialHealth){
+        if(status == StatusEffects.burning) {
+            Damage.createIncend(b.x, b.y, damage / 10f, (int)damage / 10);
```

Comment:
This should create exactly one `Fire`; `createIncend` may create multiple in a range, and that's not desirable here, especially considering damage can be 0 but splash damage can be >0.

Reply:
I changed it to `Fires.create();`